ivy_tests/test_ivy/test_frontends/test_numpy/test_matrix/test_methods.py

```diff
@@ -178,12 +178,56 @@ def test_numpy_matrix_argmax(
 
 
 # any
+@handle_frontend_method(
+        class_tree=CLASS_TREE,
+        init_tree="numpy.matrix",
+        method_name="any",
+        dtype_x_axis=helpers.dtype_values_axis(
+                available_dtypes=helpers.get_dtypes("numeric"),
+                min_num_dims=2,
+                max_num_dims=2,
+                min_dim_size=2,
+                valid_axis=True,
+                force_int_axis=True,
+                allow_neg_axes=False,
+        ),
+        to_str=st.booleans(),
+)
+def test_numpy_matrix_any(
+        dtype_x_axis,
+        to_str,
+        init_flags,
+        method_flags,
+        frontend_method_data,
+        frontend,
+):
+    input_dtype, x, axis = dtype_x_axis
+    x = _get_x_matrix(x, to_str)
+    if isinstance(axis, tuple):
+        axis = axis[0]
+    helpers.test_frontend_method(
+            init_input_dtypes=input_dtype,
+            init_flags=init_flags,
+            method_flags=method_flags,
+            init_all_as_kwargs_np={
+                    "data": x,
+                    "dtype": input_dtype[0],
+            },
+            method_input_dtypes=[],
+            method_all_as_kwargs_np={
+                    "axis": axis,
+            },
+            frontend=frontend,
+            frontend_method_data=frontend_method_data,
+    )
+
+# sum
 @handle_frontend_method(
     class_tree=CLASS_TREE,
     init_tree="numpy.matrix",
-    method_name="any",
+    method_name="sum",
     dtype_x_axis=helpers.dtype_values_axis(
-        available_dtypes=helpers.get_dtypes("numeric"),
+        available_dtypes=helpers.get_dtypes("numeric", full=False),
```

Comment:
the original default `full=True` is correct here, what I meant was to add an extra argument, for example:
```
dtype=helpers.get_dtypes("numeric", full=False)
```
so that you're able to pass it to the `method_all_as_kwargs_np` argument in the `test_frontend_method` below for testing the behaviour of the function with this argument